design-system-docs/src/_guides/using-with-rails.md

```diff
@@ -0,0 +1,315 @@
+```
+
+And finally, add the following to your `app/assets/stylesheets/application.sass.scss` entrypoint:
+
+```
+@import '@citizensadvice/design-system/scss/lib.scss';
+```
+
+If this has worked you should be able to restart your application and see the words "Hello Design System!" using the Citizens Advice brand font (Open Sans).
+
+<img src="/images/guides/using-with-rails-02.png" alt="A Rails application showing Hello Design System! and using Open Sans as the typeface"/>
+
+## Setting up an application layout
+
+The basic template generated by our `rails new` command gave us enough to get going but in order to use all the design system components we'll need to add a few things to our `application.html.erb` layout.
+
+The following is an annotated layout template including any Rails defaults which you can adapt to suit your application:
+
+```erb
+<!DOCTYPE html>
+<%%# The `no-js class` is required by the Design System styles %>
+<html class="no-js" lang="<%%= I18n.locale %>">
+  <head>
+    <meta charset="utf-8" />
+    <title>Design System guide</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%%= csrf_meta_tags %>
+    <%%= csp_meta_tag %>
+
+    <%%# Adding `media: "all"` to the stylesheet allows inline print styles to be loaded %>
+    <%%= stylesheet_link_tag "application", media: "all" %>
+
+    <%%# Pre-loading the Design System fonts can help with flash of unstyled text %>
+    <link href="/assets/fonts/open-sans-v26-latin-700.woff2" rel="preload" as="font" type="font/woff2" crossorigin="true" />
+    <link href="/assets/fonts/open-sans-v26-latin-regular.woff2" rel="preload" as="font" type="font/woff2" crossorigin="true" />
+    <link href="/assets/fonts/cads.woff" rel="preload" as="font" type="font/woff" crossorigin="true" />
+
+    <%%# This script pairs with the no-js class on the HTML element and is needed for fallback styles. %>
+    <script>document.querySelector('html').classList.remove('no-js');</script>
+    <%%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+  </head>
+  <body>
+    <%%# This ID is required when using the skip links provided by the header component %>
+    <div id="cads-main-content"><%%= yield %></div>
+  </body>
+</html>
+```
+
+Once you've done this we can also update our sample view at `app/views/home/index.html.erb` to include some layout classes.
+
+```erb
+<%%# The cads-page-content container represents the main page wrapper %>
+<div class="cads-page-content">
+  <%%# See https://citizens-advice-design-system.netlify.app/foundations/grid/ %>
+  <div class="cads-grid-container">
+    <div class="cads-grid-row">
+      <%%# Using the `md` variant here ensures the colum is responsive %>
+      <main class="cads-grid-col-md-8">
+        <h1 class="cads-page-title">Hello Design System!</h1>
+      </main>
+    </div>
+  </div>
+</div>
+```
```

Comment:
This snippet is part of the page layout component - could having it here give the impression there isn't an equivalent component? 
On the other hand, this works well being a first addition to the view while showcasing the classes

Reply:
The `PageLayout` component doesn't actually exist as part of the design system, it's more a convention we've settled on within applications themselves. Might make a good contribution if you fancy it though!